src/Components/benchmarkapps/Wasm.Performance/TestApp/Wasm.Performance.TestApp.csproj

```diff
@@ -5,7 +5,7 @@
     <IsTestAssetProject>true</IsTestAssetProject>
     <!--
       Chrome in docker appears to run in to cache corruption issues when the cache is read multiple times over.
-      Clien caching isn't part of our performance measurement, so we'll skip it.
+      Client caching isn't part of our performance measurement, so we'll skip it.
```

Comment:
How many times is app started to produce one AVG value ?
Is it cold start or warm start ?

@javiercn